cogstat/cogstat_stat.py

```diff
@@ -699,6 +699,168 @@ def variable_pair_standard_effect_size(data, meas_lev, sample=True, normality=No
+        True if possible multicollinearity was detected (VIF>10). None if the parameter was not specified.
+    sample : bool
+        True for sample descriptives, False for population estimations.
+    result : statsmodels regression result object
+        The result of the multiple regression analyses.
+
+    Returns
+    -------
+    html text
+    """
+    # TODO validate
+    import pingouin as pg
+
+    standardized_effect_size_result = '<cs_h3>' + _('Standardized effect size') + '</cs_h3>' + "\n"
+
+    # Warnings based on the results of the assumption tests
+    if normality is None:
+        standardized_effect_size_result += '\n' + '<decision>' + _('Normality could not be calculated.') + ' ' + \
+                                           _('CIs may be biased.') + '</decision>'
+    elif not normality:
+        standardized_effect_size_result += '\n' + '<decision>' + \
+                                           _('Assumption of normality violated for CI calculations.') + ' ' + \
+                                           _('CIs may be biased.') + '</decision>'
+    else:
+        standardized_effect_size_result += '\n' + '<decision>' + \
+                                           _('Assumption of normality for CI calculations met.') + '</decision>'
+
+    if homoscedasticity is None:
+        standardized_effect_size_result += '\n' + '<decision>' + _('Homoscedasticity could not be calculated.') + ' ' + \
+                                           _('CIs may be biased.') + '</decision>'
+    elif not homoscedasticity:
+        standardized_effect_size_result += '\n' + '<decision>' \
+                                           + _('Assumption of homoscedasticity violated for CI calculations.') + ' ' + \
+                                           _('CIs may be biased.') + '</decision>'
+    else:
+        standardized_effect_size_result += '\n' + '<decision>' + _('Assumption of homoscedasticity for CI '
+                                                                   'calculations met.') + '</decision>'
+
+    if multicollinearity is None:
+        standardized_effect_size_result += '\n' + '<decision>' + _('Multicollinearity could not be calculated.') + ' ' + \
+                                           _('CIs may be biased.') + '</decision>'
+    elif not multicollinearity:
+        standardized_effect_size_result += '\n' + '<decision>' \
+                                           + _('Assumption of multicollinearity violated for CI calculations.') + ' ' + \
+                                           _('CIs may be biased.') + '</decision>'
+    else:
+        standardized_effect_size_result += '\n' + '<decision>' + _('Assumption of multicollinearity for parameter '\
+                                                                   'and CI calculations met.') + '</decision>'
+
+    # Calculate effect sizes for sample or population
+    if sample:
+        pdf_result_corr = pd.DataFrame()
+        if result:
```

Comment:
do we need to check this? if result is not available, the whole function seems to be pointless (is there any other details it calculates?) And in some other lines it is not checked, which makes the solution incoherent, and pointless again. Please, double check this in the rest of the function too.